_includes/accessibility-tests/wcag-details.html

```diff
@@ -206,4 +211,9 @@
   {% assign criterion_level = "A" %}
   {% assign criterion_name = "Name, role, value" %}
   {% assign criterion_url = "https://www.w3.org/TR/WCAG21/#name-role-value" %}
+{% endif %}
+{% if item.wcag_criterion == "4.1.3" %}
+  {% assign criterion_level = "AA" %}
+  {% assign criterion_name = "Status Messages" %}
+  {% assign criterion_url = "https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#413-status-messages-aa" %}
```

Comment:
@RachelCorsino 

**issue:** We should follow the existing model for these 2.1 links
```suggestion
  {% assign criterion_url = "https://www.w3.org/TR/WCAG21/#status-messages" %}
```

Reply:
Resolved in [23c28b1](https://github.com/uswds/uswds-site/pull/2963/commits/23c28b1b772ee9faf1bc5fb99040b3d3a7fa4f45)